build.sbt

```diff
@@ -31,13 +31,14 @@ lazy val library = Project(libName, file("."))
   .settings(
     name := libName,
     scalaVersion        := "2.11.12",
-    crossScalaVersions  := Seq("2.11.12", "2.12.8"),
+    crossScalaVersions  := Seq("2.11.12", "2.12.10"),
```

Comment:
Does this mean we are removing support for Scala < 2.12.10?  Is this because they are not compatible with Play 2.7?

Reply:
No as when picking out a cross version only the major/minor version is used, i.e. `2.12`. It just updates the version used for the cross-compilation process, which we generally do if it is a patch version

Reply:
OK